src/scores.jl

```diff
@@ -472,11 +472,83 @@ function get_score_values(chain_data::ChainScoreData, score_name::String)
 end
 
 
+function district_and_plan_score_names(chain_data::ChainScoreData)::Array{String, 1}
```

Comment:
Change to `score_names`? Feels clunky to me.

Reply:
yeah, the issue is that i *only* want to extract district-level and plan-level scores (not CompositeScores) ☹️

Reply:
i've changed it to `flattened_score_names` and explained in the docstring! ("flattened" because i'm taking all the names of scores that are nested within `CompositeScores`)